processor/processor.go

```diff
@@ -0,0 +1,155 @@
+package processor
+
+import (
+	"encoding/json"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/jtblin/kube2iam/iam"
+)
+
+// RoleProcessor handles relevant logic around associating IPs with a given IAM role
+type RoleProcessor struct {
```

Comment:
`processor.RoleProcessor` is a bit redundant and a bit of a mouthful and `processor` is too generic for a package name. It seems this is more about mapping ips and roles so something around `mapper` or `mappings` may be better suited for the package name.